packages/batch-submitter/src/batch-submitter.ts

```diff
@@ -22,21 +26,40 @@ import {
   TxType,
   ctcCoder,
   EthSignTxData,
+  Address,
+  Bytes32,
 } from './coders'
 import { L2Block, BatchElement, Batch, QueueOrigin } from '.'
 
+export interface RollupInfo {
+  signer: Address
+  mode: 'sequencer' | 'verifier'
+  syncing: boolean
+  l1BlockHash: Bytes32
+  l1BlockHeight: number
+  addresses: {
+    canonicalTransactionChain: Address
```

Comment:
There is also the `stateCommitmentChain`
https://github.com/ethereum-optimism/go-ethereum/pull/59/files#diff-c426ecd2f7d247753b9ea8c1cc003f21fa412661c1f967d203d4edf8163da344R1749-R1764

Reply:
It looks like its unused, so no big deal